### ISSUE TITLE
Token Streaming

### ISSUE THREAD

**Issue by henyee** [OpenNMT/CTranslate2#1161]
Is there a way to get token streaming to work?

**Comment by guillaumekln** [comment]
Just to make sure, do you mean returning the output tokens as soon as they are generated by the model?

**Comment by henyee** [comment]
> Just to make sure, do you mean returning the output tokens as soon as they are generated by the model?

Yes, just like how ChatGPT and various LLaMA models have been made to stream the tokens.

**Comment by guillaumekln** [comment]
For reference, there is a usage example in the documentation: https://opennmt.net/CTranslate2/generation.html#token-streaming
